pages/linux/systemd-analyze.md

```diff
@@ -9,3 +9,11 @@
 - Print a tree of the time critical chain of units:
 
 `systemd-analyze critical-chain`
+
+- Create an SVG file showing when each system service started, highlighting the time that they spent on initialization:
+
+`systemd-analyze plot > {{path/to/file.svg}}`
```

Comment:
Nice output.

Reply:
Have you tried https://github.com/tldr-pages/tldr/pull/5784#discussion_r615412965?